git/diff.go

```diff
@@ -11,14 +11,14 @@ import (
 
 func gitDiff(dir string) (string, error) {
 	var out bytes.Buffer
-	cmd := exec.Command("git", "diff", "-G", "-w", "--ignore-all-space", "(^[^#])", "HEAD")
+	cmd := exec.Command("git", "diff", "-G", "(^[^#])", "-w", "--ignore-all-space", "HEAD")
 	cmd.Dir = dir
 	cmd.Stdout = &out
 	err := cmd.Run()
 	if err != nil {
 		return "", fmt.Errorf("diff in %s: %w", dir, err)
 	}
-	log.Trace().Bytes("output", out.Bytes()).Str("dir", dir).Msg("git diff")
+	log.Info().Bytes("output", out.Bytes()).Str("dir", dir).Msg("git diff")
```

Comment:
This is not very readable. Also, it doesn't make sense to use the logger for output. 
What I suggest is to add a flag to the diff subcommand which will also print the diff string, nicely formatted.

Reply:
Thanks for the input, I will change the output and add a pretty print as well. I was more dealing with GH and annotations which seem they doesn't support multiline string 👎

Reply:
Ah, I see. Perhaps just the single line summary that is printed now might be enough?